### ISSUE TITLE
CAN'T LOGIN EVEN WITH THE COOKIE METHOD

### ISSUE THREAD

**Issue by mrwhite121** [JonasNilson/idle_master_extended#314]
it just says data you entered is invalid or something like that
the main sign in button doesn't work anyway too. 
what should i do?

----

Potential solution: https://github.com/JonasNilson/idle_master_extended/issues/314#issuecomment-1179745276

**Comment by ImAndr3w** [comment]
me too

**Comment by rusudc** [comment]
happens to me too

**Comment by mateot93** [comment]
Same here, sign in doesn't react

**Comment by RhHUBz** [comment]
me 2

**Comment by TheWoodsy** [comment]
idlemanager > login > enter steam un/pw > click sign in > wait forever, does nothing
opened steam store (nit community) wanted new login > logged in, used new 2fa from phone
went idlemanager setting > harvested cookie > login worked

seems login token data was cleared at some stage recently and i had to manually log back in through browser to get the cookie method working again.

**Comment by JUPITER-cmd** [comment]
me2

**Comment by JonasNilson** [comment]
> it just says data you entered is invalid or something like that
> what should i do?

Are you using the correct cookies?
https://github.com/JonasNilson/idle_master_extended/wiki/Login-methods

Can you provide some screenshots of the Idle Master Extended information message box?

**Comment by adminerest** [comment]
Yesterday (2022-07-04) it seems that steam temporarily modified the cookies required to verify the login status.
```
Cookies needed yesterday (2022-07-04)
steamMachineAuth<SteamID>
steamRememberLogin
steamLoginSecure
```
But I feel like now should be the same as before as i tested just now

**Comment by JonasNilson** [comment]
I just experienced this issue myself. And it seems you can put in the `steamMachineAuth` cookie in place of the `steamParental` cookie:

- [image](https://user-images.githubusercontent.com/5990550/178150404-2a55d8b5-574e-4f02-92ab-17c7ae5614cd.png)

- ![image](https://user-images.githubusercontent.com/5990550/178150520-1e089595-b1b0-4936-b949-deb1f0455909.png)


@adminerest Do you have a source for that information? I would love to read up more on it

**Comment by adminerest** [comment]
I don't think i can provide any evidence for that cause I didn't save them.
I cannot guarantee that my tests are completely accurate because I forgot the combinations of the cookies I tested
but this issue must be related with those three cookies i mentioned above
so it may be works with only `steamMachineAuth` and `steamLoginSecure` or you need all three cookies
i think we need to do more tests to find out what we really need

**Comment by adminerest** [comment]
Maybe i can say a little more about that.
So about 1 day before my first comment, i wanted to run one of my script to checkout my cards stock. And i found even i refresh the page to renew the cookie, re-login, i couldn't run the script only with `steamLoginSecure` cookie also it (`steamLoginSecure`) didn't change at all. i used postman to test what was going on. i deleted all the headers except cookies and the request still work. i started to delete each cookie and found when the cookies only contains `steamMachineAuth<SteamID>`, `steamRememberLogin`, `steamLoginSecure`, i could get the page which was logged in. i tested my other script and it also had that problem. so i came to check issue here as i knew that idle only use `steamLoginSecure` as well. at the time my first comment, i could only use `steamLoginSecure` to run my script. I thought it was just a temporary modification and i didn't know whether steam would change it permanently or not.
because i didn't use `sessionid` in all my script so i dont know how this affect the `sessionid` and my tests didn't include `sessionid` as well

**Comment by FateChan** [comment]
Works Fine thanks

**Comment by M4KSuMK4** [comment]
I set everything up correctly, but the program doesn't start the calculation, and I don't know why.

**Comment by jonas-med-ett-s** [comment]
> I set everything up correctly, but the program doesn't start the calculation, and I don't know why.

What kind of messages do you get from the application?

Have you tried this? https://github.com/JonasNilson/idle_master_extended/issues/314#issuecomment-1179745276

**Comment by M4KSuMK4** [comment]
it just says data you entered is invalid or something like that

**Comment by M4KSuMK4** [comment]
https://prnt.sc/1wySbD0ZxkK8
Is it just these three, right?

**Comment by Timater** [comment]
He doesn't come in either

**Comment by FowlizM** [comment]
> it just says data you entered is invalid or something like that the main sign in button doesn't work anyway too. what should i do?
> 
> Potential solution: [#314 (comment)](https://github.com/JonasNilson/idle_master_extended/issues/314#issuecomment-1179745276)

For opera gx you can use "**sessionid**" instead of steamMachineAuth or steamparental. i hope this gets added to the interface in the next update
![image](https://github.com/JonasNilson/idle_master_extended/assets/74413584/878924f0-9902-405e-9e06-dbdbd3598707)

**Comment by Koaxz** [comment]
steamParental is missing

**Comment by Miraihi** [comment]
I always get the error when trying to log in using cookie method, but it's actually logging me in just fine when I close the settings.

**Comment by ErtasmisOlcay** [comment]
> I just experienced this issue myself. And it seems you can put in the `steamMachineAuth` cookie in place of the `steamParental` cookie:
> 
> * ![image](https://user-images.githubusercontent.com/5990550/178150404-2a55d8b5-574e-4f02-92ab-17c7ae5614cd.png)
> * ![image](https://user-images.githubusercontent.com/5990550/178150520-1e089595-b1b0-4936-b949-deb1f0455909.png)
> 
> @adminerest Do you have a source for that information? I would love to read up more on it

I have not been able to log in for a long time. I am getting the error "The data you entered is invalid, please try again". I tried again today with hope and the same error. Then I tried again with hope, turning on family protection and trying steamparental, but I still get the same error. No matter what I do, I can't find a solution.

![image](https://github.com/user-attachments/assets/de12c1c7-7025-40ad-b7aa-ae3d8b464f87)

**Comment by kyle1046** [comment]
> > I just experienced this issue myself. And it seems you can put in the `steamMachineAuth` cookie in place of the `steamParental` cookie:
> > 
> > * ![image](https://user-images.githubusercontent.com/5990550/178150404-2a55d8b5-574e-4f02-92ab-17c7ae5614cd.png)
> > * ![image](https://user-images.githubusercontent.com/5990550/178150520-1e089595-b1b0-4936-b949-deb1f0455909.png)
> > 
> > @adminerest Do you have a source for that information? I would love to read up more on it
> 
> I have not been able to log in for a long time. I am getting the error "The data you entered is invalid, please try again". I tried again today with hope and the same error. Then I tried again with hope, turning on family protection and trying steamparental, but I still get the same error. No matter what I do, I can't find a solution.
> 
> ![image](https://private-user-images.githubusercontent.com/148847185/349043610-de12c1c7-7025-40ad-b7aa-ae3d8b464f87.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MjEzMzg0NDMsIm5iZiI6MTcyMTMzODE0MywicGF0aCI6Ii8xNDg4NDcxODUvMzQ5MDQzNjEwLWRlMTJjMWM3LTcwMjUtNDBhZC1iN2FhLWFlM2Q4YjQ2NGY4Ny5wbmc_WC1BbXotQWxnb3JpdGhtPUFXUzQtSE1BQy1TSEEyNTYmWC1BbXotQ3JlZGVudGlhbD1BS0lBVkNPRFlMU0E1M1BRSzRaQSUyRjIwMjQwNzE4JTJGdXMtZWFzdC0xJTJGczMlMkZhd3M0X3JlcXVlc3QmWC1BbXotRGF0ZT0yMDI0MDcxOFQyMTI5MDNaJlgtQW16LUV4cGlyZXM9MzAwJlgtQW16LVNpZ25hdHVyZT0xOWJiYzM0NmJlZGY1YzVjOTExNDUyM2VmMmVkNjc4ZTUyOWViZjlmOWZjZjkxY2MzOTUwNTg3YTU1NDk2OWRiJlgtQW16LVNpZ25lZEhlYWRlcnM9aG9zdCZhY3Rvcl9pZD0wJmtleV9pZD0wJnJlcG9faWQ9MCJ9.z7Qbip0-NXMkRKPANBc4lN-tuy6cPAdaRTxDCD5oMV0)

i was able to get mine to work with some tickery i had to go back to "Idle Master Extended v1.8-BETA" click settings then the lock and enter the cookies and it was able to login i then went and closed the "V1.8-beta" and opened "V1.10" for a few seconds to generate the config file for it then if you go to "AppData\Local\IdleMasterExtended" you will see folders labled "IdleMasterExtended.exe_Url_******************" if you open those folders you will find one of the folders labled "1.8.0.0" inside that folder is the config file you need to copy that file and then find the folder labled "1.10.0.0" and replace the exising config file with the one you copyed at which point you can reopen "Idle Master Extended v1.10" and you will be logged in

**Comment by Crizator** [comment]
To anyone who has error "The data you entered is invalid, please try again", you take sessionID and steamLoginSecure from the wrong page cookie! Open Steam Community page and then use F12 key to look info from cookie, this is important. Cookie from all other pages don't valid for this procedure!

**Comment by Tokinish** [comment]
> To anyone who has error "The data you entered is invalid, please try again", you take sessionID and steamLoginSecure from the wrong page cookie! Open Steam Community page and then use F12 key to look info from cookie, this is important. Cookie from all other pages don't valid for this procedure!

This right here.... THANK YOU!!! <3

**Comment by mjun0906** [comment]
> > 我自己刚刚经历过这个问题。似乎你可以用 cookie 来代替 cookie：`steamMachineAuth``steamParental`
> > 
> > * ![图像](https://user-images.githubusercontent.com/5990550/178150404-2a55d8b5-574e-4f02-92ab-17c7ae5614cd.png)
> > * ![图像](https://user-images.githubusercontent.com/5990550/178150520-1e089595-b1b0-4936-b949-deb1f0455909.png)
> > 
> > 你有这些信息的来源吗？我很想阅读更多关于
> 
> 我已经很长时间无法登录了。我收到错误“您输入的数据无效，请重试”。我今天带着希望又试了一次，但还是犯了同样的错误。然后我满怀希望地再次尝试，打开家庭保护并尝试 steamparental，但我仍然收到同样的错误。无论我做什么，我都找不到解决方案。
> 
> ![图像](https://private-user-images.githubusercontent.com/148847185/349043610-de12c1c7-7025-40ad-b7aa-ae3d8b464f87.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MjY4MjMwODEsIm5iZiI6MTcyNjgyMjc4MSwicGF0aCI6Ii8xNDg4NDcxODUvMzQ5MDQzNjEwLWRlMTJjMWM3LTcwMjUtNDBhZC1iN2FhLWFlM2Q4YjQ2NGY4Ny5wbmc_WC1BbXotQWxnb3JpdGhtPUFXUzQtSE1BQy1TSEEyNTYmWC1BbXotQ3JlZGVudGlhbD1BS0lBVkNPRFlMU0E1M1BRSzRaQSUyRjIwMjQwOTIwJTJGdXMtZWFzdC0xJTJGczMlMkZhd3M0X3JlcXVlc3QmWC1BbXotRGF0ZT0yMDI0MDkyMFQwODU5NDFaJlgtQW16LUV4cGlyZXM9MzAwJlgtQW16LVNpZ25hdHVyZT02MzYzYWFlODMxOWY4ZmFmZWYxOGJjZDdlMGJjY2NlNjFiOWFhY2M5NGZiYjFiMmVmN2RiMGM5NjI2NTQ5MmE1JlgtQW16LVNpZ25lZEhlYWRlcnM9aG9zdCJ9.FRuWC7IijhLAAzODj2O2U_Pl7dvdH-mAP87-8-zcX8s)

yeah，we have the same problem。It was to be hoped that there would be new solutions

**Comment by mnogokotin** [comment]
> To anyone who has error "The data you entered is invalid, please try again", you take sessionID and steamLoginSecure from the wrong page cookie! Open Steam Community page and then use F12 key to look info from cookie, this is important. Cookie from all other pages don't valid for this procedure!

works for me

**Comment by saulob** [comment]
> To anyone who has error "The data you entered is invalid, please try again", you take sessionID and steamLoginSecure from the wrong page cookie! Open Steam Community page and then use F12 key to look info from cookie, this is important. Cookie from all other pages don't valid for this procedure!

This!! This fixed for me! That was the problem! Thanks

**Comment by Tomass2904** [comment]
buenas no me anda me tira el error de "Los datos que ha introducido no son válidos, inténtelo de nuevo" y no tengo la cookie de steam parental ni la de steamMachineAuth donde lo encontraria?? 

![Image](https://github.com/user-attachments/assets/aa9fd041-cd6e-46e4-bc44-d0125bb95120)

**Comment by IAMNOTRANA** [comment]
> To anyone who has error "The data you entered is invalid, please try again", you take sessionID and steamLoginSecure from the wrong page cookie! Open Steam Community page and then use F12 key to look info from cookie, this is important. Cookie from all other pages don't valid for this procedure!

Bruh. Why is this not mention in user guide? Thanks anyway

**Comment by PaRaDoXlI** [comment]
On the steam community opened cookies with f12->application. Entered sessionID, loginsecure. Cheked the machineauth, but there's not this name. And 'The data you entered is invalid.' I don't know, what I did wrong. I checked whole forum, and there's nothing that could help me. Please, help me

**Comment by itsccao** [comment]
> On the steam community opened cookies with f12->application. Entered sessionID, loginsecure. Cheked the machineauth, but there's not this name. And 'The data you entered is invalid.' I don't know, what I did wrong. I checked whole forum, and there's nothing that could help me. Please, help me

tldr: you can use an extensible proxy server like [krlvm/PowerTunnel](https://github.com/krlvm/PowerTunnel) to make the connection successfully.

as i read the error.log in the directory folder, i found the first error:
```13/04/2025 8:03:07 AM   CookieClient -> WebResponse = System.Net.HttpWebRequest
System.Net.WebException: The underlying connection was closed: An unexpected error occurred on a send. ---> System.IO.IOException: Unable to read data from the transport connection: A connection attempt failed because the connected party did not properly respond after a period of time, or established connection failed because connected host has failed to respond. ---> System.Net.Sockets.SocketException: A connection attempt failed because the connected party did not properly respond after a period of time, or established connection failed because connected host has failed to respond
   at System.Net.Sockets.Socket.EndReceive(IAsyncResult asyncResult)
   at System.Net.Sockets.NetworkStream.EndRead(IAsyncResult asyncResult)
   --- End of inner exception stack trace ---
   at System.Net.TlsStream.EndWrite(IAsyncResult asyncResult)
   at System.Net.PooledStream.EndWrite(IAsyncResult asyncResult)
   at System.Net.ConnectStream.WriteHeadersCallback(IAsyncResult ar)
   --- End of inner exception stack trace ---
   at System.Net.HttpWebRequest.GetResponse()
   at IdleMasterExtended.CookieClient.GetWebResponse(WebRequest request, IAsyncResult result)
```

idk if it has something to do with the internet provider i'm using but after turning on PowerTunnel, it works fine.

or you can idle each games by yourself by running a batch script, the tutorial can be found here: https://www.youtube.com/watch?v=bYyHmRDvEPk

**Comment by itsccao** [comment]
> [!NOTE]
> note that when running the batch script above, it won't automatically update game cards. it will only update game cards after you stop running the steam-idle.exe

**Comment by f0rWard33** [comment]
> To anyone who has error "The data you entered is invalid, please try again", you take sessionID and steamLoginSecure from the wrong page cookie! Open Steam Community page and then use F12 key to look info from cookie, this is important. Cookie from all other pages don't valid for this procedure!

Thanks its working :)

**Comment by daviddprtma** [comment]
> To anyone who has error "The data you entered is invalid, please try again", you take sessionID and steamLoginSecure from the wrong page cookie! Open Steam Community page and then use F12 key to look info from cookie, this is important. Cookie from all other pages don't valid for this procedure!

you're saving my life dude. this answer should be mark as the best answer for the new version👍😍

**Comment by jopapopa24124** [comment]
If someone have this issue, now that "parental" or "steammachineauth" not exist, noww its calld "browserid" use it in parental field
